src/nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -884,6 +897,22 @@ def apply(
         # Filter all_weight_params and by excluding nodes that should remain in their original floating-point precision
         all_weight_params = list(filter(lambda w_params: w_params.compression_config is not None, all_weight_params))
 
+        return all_weight_params, statistics
+
+    def apply(
+        self,
+        model: TModel,
+        graph: NNCFGraph,
+        statistic_points: Optional[StatisticPointsContainer] = None,
+        dataset: Optional[Dataset] = None,
+    ) -> TModel:
+        self.set_backend_entity(model)
+
+        # Get processed weight compression parameters ready for compression
+        all_weight_params, statistics = self.get_processed_weight_compression_parameters(
```

Comment:
I got from the docstring that there are three elements in the output tuple. But looks like it is only two. Please update the docstring